entity-framework/core/modeling/data-seeding.md

```diff
@@ -86,3 +87,14 @@ Depending on the constraints of your deployment the initialization code can be e
 * Deploying the initialization app with the main app, invoking the initialization routine and disabling or removing the initialization app.
 
 This can usually be automated by using [publish profiles](/aspnet/core/host-and-deploy/visual-studio-publish-profiles).
+
+<a name="use-seeding-method"></a>
+
+### Configuration options `UseSeeding` and `UseAsyncSeeding` methods
+
+EF 9 introduced `UseSeeding` and `UseAsyncSeeding` methods, which provide convenient way of configuring custom initialization logic. These methods can be set up in the [options configuration step](/ef/core/dbcontext-configuration/#dbcontextoptions). Here is an example analogous to the custom initialization showed above:
```

Comment:
Also mention that one of these methods is called when `Migrate` is called or when `dotnet ef database update` is run, even if no migrations were applied